lib/improver/tests/ensemble_calibration/ensemble_calibration/test_ApplyCoefficientsForEnsembleCalibration.py

```diff
@@ -271,245 +275,169 @@ def test_unmatching_y_axis(self):
-                "realization", iris.analysis.MEAN))
-        predicted_mean, forecast_predictors = (
-            self.plugin._get_calibrated_forecast_predictors_mean(
+        """Test that the expected values for the location parameter are
+        calculated when using the ensemble mean. These expected values are
+        compared to the results when using the ensemble realizations to ensure
+        that the results are similar."""
+        location_parameter = (
+            self.plugin._calculate_location_parameter_from_mean(
                 self.optimised_coeffs))
         self.assertCalibratedVariablesAlmostEqual(
-            predicted_mean, self.expected_calibrated_predictor_mean)
-        self.assertCalibratedVariablesAlmostEqual(
-            forecast_predictors.data, expected_forecast_predictors.data)
+            location_parameter, self.expected_loc_param_mean)
+        assert_array_almost_equal(
+            location_parameter,
+            self.expected_loc_param_statsmodels_realizations,
+            decimal=0)
+        assert_array_almost_equal(
+            location_parameter,
+            self.expected_loc_param_no_statsmodels_realizations,
+            decimal=0)
 
 
-class Test__get_calibrated_forecast_predictors_realizations(
+class Test__calculate_location_parameter_from_realizations(
         SetupCoefficientsCubes, EnsembleCalibrationAssertions):
 
-    """Test the _get_calibrated_forecast_predictors_realizations method."""
+    """Test the _calculate_location_parameter_from_realizations method."""
 
     @ManageWarnings(
         ignored_messages=["Collapsing a non-contiguous coordinate."])
     def setUp(self):
-        """Setup cubes and sundries for testing calibration of mean."""
+        """Set-up coefficients and plugin for testing."""
         super().setUp()
 
-        self.forecast_vars = self.current_temperature_forecast_cube.collapsed(
-            "realization", iris.analysis.VARIANCE)
-
         self.plugin = Plugin()
         self.plugin.current_forecast = self.current_temperature_forecast_cube
-        self.expected_calibrated_predictor_statsmodels_realizations = (
-            self.expected_calibrated_predictor_statsmodels_realizations.
-            flatten())
-        self.expected_calibrated_predictor_no_statsmodels_realizations = (
-            self.expected_calibrated_predictor_no_statsmodels_realizations.
-            flatten())
 
     @ManageWarnings(
         ignored_messages=["Collapsing a non-contiguous coordinate."])
     def test_with_statsmodels(self):
-        """
-        Test that the expected values are returned by this function when using
-        statsmodels.
-        """
+        """Test that the expected values for the location parameter are
+        calculated when using the ensemble realizations with statsmodels.
+        These expected values are compared to the results when using the
+        ensemble realizations to ensure that the results are similar."""
```

Comment:
Typo: '...compared to the results when using the ensemble **mean** to ensure...'

You could also mention that the result is compared to the expected result for the no-statsmodels case as well.

Reply:
I've corrected this as suggested.